sdh-proxy.c

```diff
@@ -136,6 +145,36 @@ unsigned short int * udp_get_port( const short unsigned int pktlen, const u_char
 
 }
 
+/** 
+ * function to log some statistical Ouitput to a logfile
+ * This is usefull to get somewhat of information / statistical logging.
+ *
+
+ *
+ **/
+void writeLogStats()
+{
+    // don't spam! just write the log once every X seconds
+    if((logtimer + 10) < (int)time(NULL)) {
+      FILE * fp;
+      fp = fopen(logstatfile, "w+");
+      if (fp) 
+      {
+        // General Stats
+        fprintf(fp,"## GENERAL\nRX:%ld\nTX:%ld\nDROP:%ld\n## PORT STATS\n",(long) pkt_rx, (long) pkt_tx, (long) pkt_drop);
```

Comment:
I choose this way of logfile because it'S still human readable. For pushing it to influxDB I used a sperate bash Script to convert this into a one line insert every 10s.